test/e2e/clustertask/start_test.go

```diff
@@ -0,0 +1,114 @@
+// +build e2e
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clustertask
+
+import (
+	"testing"
+
+	"github.com/AlecAivazis/survey/v2/terminal"
+	"github.com/Netflix/go-expect"
+	"github.com/tektoncd/cli/test/builder"
+	"github.com/tektoncd/cli/test/cli"
+	"github.com/tektoncd/cli/test/framework"
+	"github.com/tektoncd/cli/test/helper"
+
+	"gotest.tools/v3/assert"
+	is "gotest.tools/v3/assert/cmp"
+	"gotest.tools/v3/icmd"
+	knativetest "knative.dev/pkg/test"
+)
+
+const (
+	tePipelineGitResourceName = "skaffold-git"
+)
+
+func TestClusterTaskInteractiveStartE2E(t *testing.T) {
```

Comment:
There should also be a test here for `volumeClaimTemplateFile` with `-w`, but I can add that in a follow up pr.